src/cmd/run.js

```diff
@@ -80,6 +81,11 @@ export default async function run(
   // Create an alias for --pref since it has been transformed into an
   // object containing one or more preferences.
   const customPrefs = { ...pref };
+
+  // Create an alias for --chromium-pref since it has been transformed into an
+  // object containing one or more preferences.
+  const customChromiumPrefs = { ...chromiumPref };
```

Comment:
I see that you basically copied the previous few lines. The logic there did not completely make sense, I've sent a PR to fix that up: https://github.com/mozilla/web-ext/pull/3218

Let's simplify to the following:

```suggestion
  const customChromiumPrefs = chromiumPref;
```